lib/link_thumbnailer/scrapers/opengraph/image.rb

```diff
@@ -20,7 +21,11 @@ def value
           end
 
           def model
-            nodes.map { |n| modelize(n, n.attributes['content'].to_s) }
+            nodes.map do |n|
+              if LinkThumbnailer::URI.new(n.attributes['content']).valid?
```

Comment:
exactly :) With some unit testing and it should be good :)

probably can refactor the code to something like this:

```ruby
nodes.map { |n| LinkThumbnailer::URI.new(n.attributes['content']) }.keep_if(&:valid?)
```